state-chain/chains/src/mocks.rs

```diff
@@ -1,4 +1,4 @@
-#![cfg(debug_assertions)]
+#![cfg(feature = "std")]
```

Comment:
Seems that the mocks are inconsistently enabled. Sometimes they use `feature = "std"` and somethings `cfg(debug_assertions)`, probably worth making it consistent at the very least @dandanlen.